formapp/templates/users.html

```diff
@@ -75,6 +75,7 @@ <h1>User accounts</h1>
     let users = "{{ users }}".replaceAll("&#39;", "'").replaceAll("(", "[").replaceAll(")", "]").replaceAll("b'", "'").replaceAll("None", null);
     users = eval(users);
     let permNames = ['Teacher', 'Mod', 'Student', 'Guest'];
+    let tagNames = ['12th/Senior', '11th/Junior'];
```

Comment:
Why is this here?

Reply:
This went along with the other code you marked in users.html, which are both old code that I forgot to delete.